website/source/guides/advanced-scheduling/affinity.html.md

```diff
@@ -0,0 +1,231 @@
+---
+layout: "guides"
+page_title: "Affinity"
+sidebar_current: "guides-advanced-scheduling"
+description: |-
+   The affinity stanza allows operators to express placement preference for a set of nodes. Affinities may be expressed on attributes or client metadata.  
+---
+
+# Expressing Job Placement Preferences with Affinities
+
+The [affinity][affinity-stanza] stanza allows operators to express placement preferences for their jobs on particular types of nodes. It is important to remember that there is a key difference between the [constraint][constraint] stanza and the affinity stanza: while the `constraint` stanza strictly filters where jobs are run based on [attributes][attributes] and [client metadata][client-metadata], the `affinity` stanza is more flexible and allows the job to run on resources that fit best after making an attempt to match the Nomad operator's specifications. This is done in conjunction with scoring based on the Nomad scheduler's bin packing algorithm which you can read more about [here][scheduling].
+
+## Reference Material
+
+- The [affinity][affinity-stanza] stanza documentation
+- [Scheduling][scheduling] with Nomad
+
+## Estimated Time to Complete
+
+20 minutes
+
+## Challenge
+
+Think of a scenario where a Nomad operator needs the flexibility to express placement preferences for a critical job but still have the scheduler run the job on the most appropriate nodes anywhere if the desired resources are not available.
+
+## Solution
+
+Specify an affinity with the proper [weight][weight] so that the Nomad scheduler
+can find the best nodes on which to place your job. The weight of the affinity
+will be factored in with the results of the Nomad scheduler's bin packing algorithm (which is used to optimize the resource utilization and density of applications) to find the best fit.
```

Comment:
The affinity weight will be used as scoring criteria, along with other factors like the bin packing algorithm